Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "sway-zed-extension"
+version = "0.1.0"
+edition = "2024"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+
```

nit: could have bit more details:
```suggestion
description = "Sway language extension for Zed editor"
license = "Apache-2.0 OR MIT"
repository = "https://github.com/FuelLabs/sway-zed-extension"

```